### DISCUSSION TITLE
Increase input area decimal points resolution.

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#27792] [Ideas] [answered]
# Background
Currently, osu! in game tablet settings is using 0 decimal points resolution for tablet settings (except `aspect ratio` is 2 decimal points), which is hard to customize, and the value shown is hard to tell apart.

For example, compare these two image with settings `Conform with current aspect ratio` on 16:9 screen (1.77777777777... -> 1.78 shown on osu!)

| Width 29, Height 16 |  Width 28, Height 16 |
| --- | --- |
| ![image](https://github.com/ppy/osu/assets/7066150/39b05204-0a91-4f92-9d1c-1c06acd7b76a) | ![image](https://github.com/ppy/osu/assets/7066150/5e8e24af-fed2-45b6-aea5-e375bd458d7c) |

Those two have same height, which are 16, but the width is different. The way to do this is to manually slide the width value from 16 **on the right** to 16 **on the left**. Thus, the sliders have some decimal points in mind, but doesn't show the value up, which is hard for user to tell apart the values. 
*Using arrow key increase/decrease the value by 4.0, thus useless to fine-tuning the value.* 

# Idea 
- Shown additional decimal points on tablet settings.
- Using arrow keys should increase / decrease the value by 0.1 or 0.01 for tablet settings (or have key combination to do so).

Bonus
- Have using input the slider value for slider (this may be applied to other sliders as well, such as a setting `1` on `combo to mute` should not be that hard like what currently is.)

# Additional information
For `OpenTabletDriver` user have to put the value using keyboard.
![image](https://github.com/ppy/osu/assets/7066150/edbd3416-9f5e-4e54-833c-54de1456ee7d)

**Comment by SupDos** [comment, marked as answer]
I believe the devs are aware of the need for manual number input in sliders, https://github.com/ppy/osu/discussions/16932

For now, if you really want to you can edit the values manually in `input.json`, or just continue using OTD externally instead of the integrated OTD, until this feature is added
